example/90-shoot.yaml

```diff
@@ -282,7 +283,19 @@ spec:
   #   backend: bird
   #   typha:
   #     enabled: true
-  # See also: https://github.com/gardener/gardener/blob/master/docs/proposals/03-networking.md
+  # See also: https://github.com/gardener/gardener-extension-networking-calico/blob/master/docs/usage-as-end-user.md
+  # Example configuration for Cilium
+  # type: cilium
+  # pods: 100.96.0.0/11
+  # nodes: 10.250.0.0/16
+  # services: 100.64.0.0/13
+  # providerConfig:
+  #   apiVersion: cilium.networking.extensions.gardener.cloud/v1alpha1
+  #   kind: NetworkConfig
+  #   hubble:
+  #     enabled: true
+  # See also: https://github.com/gardener/gardener-extension-networking-cilium/blob/master/docs/usage-as-end-user.md
+  # For networking extensibility see also: https://github.com/gardener/gardener/blob/master/docs/proposals/03-networking-extensibility.md
```

Comment:
Can we rather make this whole section agnostic, similar to https://github.com/gardener/gardener/blob/c2237bf05b678413148a42574d1e44a34f1d43b8/example/90-shoot.yaml#L12-L28, to prevent having double-maintenance?